### PR TITLE
Add info about upgrading to 5.15 and image of web UI warning

### LINKED ISSUE #1945: Upgrading doc should describe behavior when upgrading to new limits in 5.15
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

Upgrading documentation describes expected behavior and considerations when upgrading from >= 5.14.2 to new free usage limits in 5.15.0

## Current Behavior

Upgrading documentation does not describe implications of new free usage limits for installations which exceed 100 entities.

## Possible Solution

Add a new section to the upgrading doc.

## Context

We've documented changes in the free usage limit on [Discourse](https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156) as well as the [Sensu blog](https://blog.sensu.io/one-year-of-sensu-go). 

When upgrading to 5.15, existing unlicensed instances with more than 100 entities will continue to monitor those entities, but attempts to create any new entities via the HTTP API or sensuctl will return the following message:

> This functionality requires a valid Sensu Go license with a sufficient entity limit. To get a valid license file, arrange a trial, or increase your entity limit, contact Sales.

Connections from new agents will fail with a log message like this one:

```
{"component":"agent","error":"handshake failed with status 402","level":"error","msg":"reconnection attempt failed","time":"2
019-11-20T05:49:24-07:00"}   
```

In the Web UI, operators will see the following message when the entity limit is reached:

2019-11-20 at 5 53 AM

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add section in upgrade.md to describe how 100-entity limit will affect users with unlicensed instances with more than 100 entities.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1945

## Review Instructions
I also replaced the Discourse link with the blog announcement. The Discourse post described a 1000-entity limit from May (https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156) -- I couldn't find anything about the 5.15 limit of 100 entities on Discourse (maybe I'm overlooking it?)

### CODE REVIEW

**Comment by hillaryfraley** [comment]
Re: review instructions note, I replaced the Discourse link in https://github.com/sensu/sensu-docs/pull/1947 instead.

File: content/sensu-go/5.15/installation/upgrade.md [hunk 1]
```diff
@@ -34,12 +35,33 @@ sudo service sensu-backend restart
...
+As of Sensu Go 5.15.0, Sensu's free entity limit is 100 entities. All [commercial features][33] are available for free in the packaged Sensu Go distribution up to an entity limit of 100.
+
+When you upgrade to 5.15.0, if your existing unlicensed instance has more than 100 entities, Sensu will continue to monitor those entities. However, if you try to create any new entities via the HTTP API or sensuctl, you will see the following message:
+
+`This functionality requires a valid Sensu Go license with a sufficient entity limit. To get a valid license file, arrange a trial, or increase your entity limit, contact Sales.`
+
+Connections from new agents will fail and result in a log message like this:
+
+{{< highlight shell >}}
+{"component":"agent","error":"handshake failed with status 402","level":"error","msg":"reconnection attempt failed","time":"2
```
Comment (cwjohnston) [inline content/sensu-go/5.15/installation/upgrade.md]: Please remove the line break inside the highlight block so that the log message is contiguous.

**Review by cwjohnston — APPROVED** [review]
Approved with one minor change request 👍

File: content/sensu-go/5.15/installation/upgrade.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.15/installation/upgrade.md]: Done! Thank you.

**Pull request merged**